package.json

```diff
@@ -950,25 +950,68 @@
                                 }
                             },
                             "platform": {
-                                "anyOf": [
+                                "oneOf": [
```

Comment:
anyOf is still the appropriate condition instead of oneOf; looks like JSON schema treats the two "string" type declarations (if "linux" matches both { "type": "string" } and { "type": "string", "enum": ["linux"] } as mutually exclusive. So specifying "linux" for "os" will give you a syntax warning in your tasks.json, since "linux" matches both the enum type and the general string type, but oneOf says it should only ever match one condition.